CMakeLists.txt

```diff
@@ -29,9 +29,11 @@ option(use_libclang    "Add support for libclang parsing." OFF)
 option(static_libclang "Link to a statically compiled version of LLVM/libclang." OFF)
 option(win_static      "Link with /MT in stead of /MD on windows" OFF)
 option(english_only    "Only compile in support for the English language" OFF)
-option(force_qt4       "Forces doxywizard to build using Qt4 even if Qt5 is installed" OFF)
 option(enable_coverage "Enable coverage reporting for gcc/clang [development]" OFF)
 
+set(force_qt CACHE INTERNAL "Forces doxywizard to build using the specified major version, this can be Qt5 or Qt6")
```

Comment:
When removing the `force_qt` option, how does one force the Qt version to use from the command line?

Reply:
Only the `force_qt4`  is gone, with `force_qt` one can do:
```
-Dforce_qt=Qt5
```
or
```
-Dforce_qt=Qt6
```

or did I misunderstand the question?

Reply:
No, I indeed see that this works. Was just expecting the option to show up in the list when using `cmake -L` like before.

Reply:
Indeed I was hoping for it as well  but didn't find a way to do it.